app/build.gradle

```diff
@@ -50,7 +50,7 @@ android {
     }
     packagingOptions {
         resources {
-            excludes += ['META-INF/AL2.0', 'META-INF/LGPL2.1']
+            excludes += 'META-INF/*'
```

Comment:
Lots of new files are included in META-INF with new okta management sdk version. Everything in app/ is for sample app and E2E tests, and has nothing to do with SDKs.